packages/admin/src/pages/super-admin-dashboard/user/[id]/change-roles.test.tsx

```diff
@@ -81,48 +87,36 @@ describe('Edit role page', () => {
     ).toBeChecked();
     expect(
       screen.getByText('Administrator').parentElement?.previousSibling
-    ).not.toBeChecked();
+    ).toBeChecked();
     expect(
       screen.getByText('Applicant').parentElement?.previousSibling
-    ).not.toBeChecked();
+    ).toBeChecked();
   });
 
-  test('Should redirect to change department page if new Admin user', async () => {
+  test('Should redirect to change department page as User who is being newly promoted to ADMIN', async () => {
     mockParseBody.mockResolvedValue({ newUserRoles: ['3', '4'] });
-    mockGetUserById.mockReturnValue({
-      firstName: 'john',
-      lastName: 'm',
-      organisationName: 'tco',
-      emailAddress: 'superAdmin@and.digital',
-      roles: [getMockRoles()[0], getMockRoles()[1]],
-      department: null,
-      created: 'NULL',
-    });
-
+    mockGetUserById.mockResolvedValue(
+      getMockUser([getMockRoles()[0], getMockRoles()[1]])
+    );
     const getDefaultContext = () => ({
       params: { id: '1' },
       req: { method: 'POST' },
     });
     const result = await getServerSideProps(getContext(getDefaultContext));
     expect(result).toEqual({
       redirect: {
-        destination: '/super-admin-dashboard/user/1/change-department',
+        //new roles query param passed in to be used in change department page
+        destination: `/super-admin-dashboard/user/1/change-department?newRoles=1,2,3,4`,
         statusCode: 302,
       },
     });
   });
 
-  test('Should redirect to account page as User who is an Applicant', async () => {
+  test('Should redirect to account overview page as User who is being demoted from ADMIN to APPLICANT', async () => {
     mockParseBody.mockResolvedValue({ newUserRoles: ['1', '2'] });
-    mockGetUserById.mockResolvedValue({
-      firstName: 'john',
-      lastName: 'm',
-      organisationName: 'tco',
-      emailAddress: 'superAdmin@and.digital',
-      roles: [getMockRoles()[0], getMockRoles()[1]],
-      department: null,
-      created: 'NULL',
-    });
+    mockGetUserById.mockResolvedValue(
+      getMockUser([getMockRoles()[0], getMockRoles()[1], getMockRoles()[2]])
```

Comment:
it doesn't hugely matter here, but you can use array methods to save yourself some typing when manipulating test data like this

```js
      getMockUser(getMockRoles().slice(0, 3))
```

or

```js
      getMockUser(getMockRoles().filter(role => role.id !== '4'))
```

will get you the same result